library/src/scripts/headers/titleBarStyles.ts

```diff
@@ -92,6 +92,10 @@ export const titleBarVariables = useThemeCache(() => {
         },
     });
 
+    const titleBarNavAlignment = makeThemeVars("NavAlignment", {
+        justifyContent: "flex-start",
+    });
+
```

Comment:
- `NavAlignment` should be `navAlignment` for consistency.
- We variable should probably be `alignment: 'left' as 'left' | 'center'`. From their we can translate it into a flex justification.

When making variables it's best not to couple them to the current exact implementation. In the future, multiple properties might need to get set (not just the flex justification) or it might not use flex at all. By using a more generic variable name we are freerer to change that implementation in the future.